homeassistant/components/vallox/__init__.py

```diff
@@ -309,6 +310,12 @@ async def async_handle(self, call: ServiceCall) -> None:
             _LOGGER.error("Service not implemented: %s", service_details.method)
             return
 
+        _LOGGER.warning(
```

Comment:
We create a repairs issue for these cases nowadays too.

Reply:
Thank you for the example. Implemented.